### ISSUE TITLE
Location reported as coming from cell no matter what

### ISSUE THREAD

**Issue by olejorgenb** [mendhak/gpslogger#997]
After upgrading to android 12, the details view claims the location come from cell towers. Even when I've turned off both cell and passive locations as providers. The reported accuracy is very good though and the log prints attempt at getting gps fox so I suspect the real source is GPS.

Version 106

**Comment by mendhak** [comment]
When you turned off cell and passive, did you also restart logging?  I think the changes don't take effect till you restart the logging.

**Comment by olejorgenb** [comment]
Yes. (verified again now). Testing further I see that it sometimes say GPS for a short while, then the display refresh a few times (much faster than the log interval) changing to cell (sometimes going back to GPS I think). In the log it says it's discarding points due to the accuracy filter when this is going on. I also tried to turn of the Google "improved" location service.

Samsung Galaxy 21.
